cheroot/test/helper.py

```diff
@@ -40,7 +41,7 @@ class CherootWebCase(webtest.WebCase):
     scheme = 'http'
 
     available_servers = {
-        'wsgi': cheroot.server.WSGIServer,
+        'wsgi': cheroot.wsgi.WSGIServer,
```

Comment:
At the end of the `wsgi.py` file, `WSGIServer` class was defined as an alias of the `Server` class. So both are right.

```
# compatibility aliases
globals().update(
    WSGIServer=Server,
    WSGIGateway=Gateway,
    WSGIGateway_u0=Gateway_u0,
    WSGIGateway_10=Gateway_10,
    WSGIPathInfoDispatcher=PathInfoDispatcher,
)
```